packages/airnode-validator/src/commands/validateCmd.ts

```diff
@@ -27,23 +26,12 @@ const args = yargs(hideBin(process.argv))
   })
   .parseSync();
 
-let templatePath: string | null, version: string;
-// eslint-disable-next-line prefer-const
-[templatePath, version] = args.template.split('@');
+let res: Result;
 
-if (templates[templatePath.toLowerCase() as keyof typeof templates]) {
-  templatePath = utils.getPath(templates[templatePath.toLowerCase() as keyof typeof templates], messages, version);
-} else if (version) {
-  messages.push(logger.warn('Version argument will be ignored when validating provided template file'));
-}
-
-let res: Log[] | Result;
-
-if (templatePath) {
-  res = validate(args.specification, templatePath, args.secrets);
-  res.messages.push(...messages);
+if (utils.parseTemplateName(args.template, messages)) {
```

Comment:
Because in validate command in `template` argument can be name of the template or directly path to the template itself, so we need to check first if user entered template name and if not its probably a path to the template file. While in the convert command in `template` argument can be only path to the template and in `from` and `to` arguments can be only a template name, so if there is anything else we can throw.